### PR TITLE
feat: event normalization for 'esl:refresh' and ESLImage, ESLMedia components

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

File: src/modules/esl-media/core/esl-media.ts [hunk 1]
```diff
@@ -44,6 +43,21 @@ export class ESLMedia extends ESLBaseElement {
...
+  @prop('esl:media:error') public ERROR_EVENT: string;
+  /** Event to dispatch when player play */
+  @prop('esl:media:play') public PLAY_EVENT: string;
+  /** Event to dispatch when player paused */
+  @prop('esl:media:paused') public PAUSED_EVENT: string;
+  /** Event to dispatch when player ended */
+  @prop('esl:media:ended') public ENDED_EVENT: string;
+  /** Event to dispatch when player detached */
+  @prop('esl:media:detached') public DETACHED_EVENT: string;
+  /** Event to dispatch when layer paused by another instance in group */
```
Comment (fshovchko) [inline src/modules/esl-media/core/esl-media.ts:58]:
```suggestion
  /** Event to dispatch when player paused by another instance in group */
```

File: src/modules/esl-media/core/esl-media.ts [hunk 2]
```diff
@@ -44,6 +43,21 @@ export class ESLMedia extends ESLBaseElement {
     'controls'
   ];
 
+  /** Event to dispatch on ready state */
+  @prop('esl:media:ready') public READY_EVENT: string;
+  /** Event to dispatch on error state */
+  @prop('esl:media:error') public ERROR_EVENT: string;
+  /** Event to dispatch when player play */
```
Comment (fshovchko) [inline src/modules/esl-media/core/esl-media.ts:50]:
```suggestion
  /** Event to dispatch when player plays */
```

**Review by fshovchko — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Review by nattallius — APPROVED** [review]

File: src/modules/esl-media/core/esl-media.ts [hunk 3]
```diff
@@ -44,6 +43,21 @@ export class ESLMedia extends ESLBaseElement {
...
+  /** Event to dispatch when player plays */
+  @prop('esl:media:play') public PLAY_EVENT: string;
+  /** Event to dispatch when player paused */
+  @prop('esl:media:paused') public PAUSED_EVENT: string;
+  /** Event to dispatch when player ended */
+  @prop('esl:media:ended') public ENDED_EVENT: string;
+  /** Event to dispatch when player detached */
+  @prop('esl:media:detached') public DETACHED_EVENT: string;
+  /** Event to dispatch when player paused by another instance in group */
+  @prop('esl:media:mangedpause') public MANAGED_PAUSE_EVENT: string;
```
[the remark this answers is not in the document]
Comment (ala-n) [reply src/modules/esl-media/core/esl-media.ts:59]: Ou ou ou, nice catch!
Thanks a lot, @nattallius 
I'd say it is the right moment to fix it now

```suggestion
  @prop('esl:media:managedpause') public MANAGED_PAUSE_EVENT: string;
```

**Pull request merged**